### PR TITLE
Update src/library/Form/styled

### PR DESCRIPTION
**Pull request by tx-michael**

### CODE REVIEW

**Comment by tx-michael** [comment]
to remove styling for a11y

File: src/library/Form/styled.js [hunk 1]
```diff
@@ -67,7 +67,7 @@ export const FormFieldsetRoot = styled('fieldset')(
       ...componentStyleReset(baseTheme),
 
       border: '1px solid transparent',
-      borderTopColor: theme.FormFieldset_borderColor,
+      // borderTopColor: theme.FormFieldset_borderColor,
```
Comment (tx-steven) [inline src/library/Form/styled.js:70]: I think with the current formula we would keep this as is and just change this on our theme object. 
Something like `FormFieldset_borderColor: 'transparent'`

Comment (tx-michael) [reply src/library/Form/styled.js:70]: 👍

Comment (tx-nestor) [reply src/library/Form/styled.js:70]: That's correct. Prefer to explicitly unset it to either `undefined` or `transparent`.

**Pull request closed without merging**